src/stores/__tests__/FluxMapStore-test.js

```diff
@@ -1,116 +0,0 @@
-/**
```

Comment:
If you don't mind, I think now FluxReduceStore is basically untested. This indirectly tested it. Could you just translate this as a test file for FluxReduceStore?